.github/workflows/main.yml

```
@@ -385,6 +385,7 @@ jobs:
       matrix:
         workspace:
           - examples/counter
+          - examples/composition
```

How about adding something like (**_untested_**):
```suggestion
        include:
          - workspace: examples/counter
             dev-mode: true
          - workspace: examples/composition
             dev-mode: false
```

---

Fixed by running all examples using the local prover.